scrapy/core/downloader/__init__.py

```diff
@@ -194,6 +194,6 @@ def close(self):
 
     def _slot_gc(self, age=60):
         mintime = time() - age
-        for key, slot in self.slots.items():
+        for key, slot in list(self.slots.items()):
```

Comment:
I prefer list :) There is no meaningful speed or memory difference, and `list` makes code more clear. I like to use tuples when values have special meaning depending on their position - e.g. a point with x,y coordinates or RGBA colors; lists are for cases where elements are all of the same kind.